File: cpp/Program/Shirt.cpp
#include <iostream>
#include <string>

using namespace std;

class Shirt : public Clothing
{
private:
    string color;
    string sleeveType;

public:
    Shirt(){
    }

    void setColor(string color){
        this->color = color;
    }

    string getColor(){
        return this->color;
    }

    void setSleeveType(string sleeveType){
        this->sleeveType = sleeveType;
    }

    string getSleeveType(){
        return this->sleeveType;
    }

    ~Shirt(){
    }
};

File: python/Program/Shirt.py
from Clothing import Clothing

class Shirt(Clothing):
    def __init__(self):
        super().__init__()
        self.__color = ""
        self.__sleeve_type = ""

    def set_color(self, color):
        self.__color = color

    def get_color(self):
        return self.__color

    def set_sleeve_type(self, sleeve_type):
        self.__sleeve_type = sleeve_type

    def get_sleeve_type(self):
        return self.__sleeve_type